CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 # Unreleased
 
-Nothing Yet!
+# Version 0.21.2 (2024-08-28)
+
+This patch release fixes several bugs and provides some nice quality of life improvements. These fixes are largely related to our installers, shell, powershell, and homebrew as well as managing some papercuts in the `dist init` command and error handling.
```

Comment:
I like that you can tell who wrote what tests by whether "Homebrew" is capitalized